exercises/go-2-a/.meta/start_1.go

```diff
@@ -0,0 +1,34 @@
+package go2a
+
+import (
+	"fmt"
+	"strconv"
+)
+
+func concat(description string, str string) string {
+	if description != "" {
+		return description + " or " + str
+	}
+
+	return description + str
```

Comment:
if description == "" {
return str
}
return description + " or " + str

Reply:
You probably meant:

```go
	if description == "" {
		return description + str
	}

	return description + " or " + str
```

Reply:
Right :)